plugins/org.jkiss.dbeaver.model/src/org/jkiss/dbeaver/model/sql/SQLUtils.java

```diff
@@ -1207,4 +1207,57 @@ public static String identifierToCanonicalForm(
         actualIdentifierString = forceUnquotted ? unquottedIdentifier : dialect.getQuotedIdentifier(unquottedIdentifier, true, false);
         return actualIdentifierString;
     }
+
+    public static String compact(String sql, SQLDialect sqlDialect) {
```

Comment:
This function will greatly benefit from tests and a short Javadoc that describes its purpose. Please also specify `@NotNull` and `@Nullable` annotations where applicable.